PowerEditor/src/MISC/Common/Common.cpp

```diff
@@ -1826,12 +1827,16 @@ struct GetAttrExParamResult
 DWORD WINAPI getFileAttributesExWorker(void* data)
 {
 	GetAttrExParamResult* inAndOut = static_cast<GetAttrExParamResult*>(data);
+	::SetLastError(NO_ERROR);

```

Comment:
@donho 

No, it is ok, the WINAPI error codes are per-thread ([Ref.](https://learn.microsoft.com/en-us/windows/win32/api/errhandlingapi/nf-errhandlingapi-setlasterror)): `"The last-error code is kept in thread local storage so that multiple threads do not overwrite each other's values."`

And resetting before the GetFileAttributesEx calling is necessary because of `"Most (WINAPI) functions call SetLastError(Ex) ONLY when they fail. However, some system functions call SetLastError(Ex) under conditions of success;"`. So without the reset, it can contain a misleading previously set non-zero value even after a successful GetFileAttributesEx call...